src/main/java/com/rental/rentalApp/controller/PaymentMethodController.java

```diff
@@ -0,0 +1,37 @@
+package com.rental.rentalApp.controller;
+
+import com.rental.rentalApp.entities.*;
+import com.rental.rentalApp.repositories.PaymentMethodRepository;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+
+@RestController
+@RequestMapping("rental-app/api/payment-method")
+public class PaymentMethodController {
+
+    private final PaymentMethodRepository paymentMethodRepository;
+
+    public PaymentMethodController(PaymentMethodRepository paymentMethodRepository) {
+        this.paymentMethodRepository = paymentMethodRepository;
+    }
+
+    @GetMapping("get-payment-methods")
```

Comment:
in light of above suggestions

```suggestion
    @GetMapping("")
```